examples/1d/reconstruct_torch.py

```diff
@@ -58,16 +59,16 @@ def generate_harmonic_signal(T, num_intervals=4, gamma=0.9, random_state=42):
 # Let’s take a look at what such a signal could look like.
 
 T = 2 ** 13
-x = torch.from_numpy(generate_harmonic_signal(T))
+x = torch.from_numpy(generate_harmonic_signal(T)).to(device)
```

Comment:
Maybe just move this call to just before the `scattering` call (so have `x = x.to(device)` on the line before)?